front/app/components/VoteControl/VotingDisabled.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
I'm sure it can be improved. Feedback welcome!

I tested and the old code that I fixed broke the test, so good in that respect 👍